CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add `Py::setattr` method. [#2009](https://github.com/PyO3/pyo3/pull/2009)
 - Add `PyCapsule`, exposing the [Capsule API](https://docs.python.org/3/c-api/capsule.html#capsules). [#1980](https://github.com/PyO3/pyo3/pull/1980)
+- `#[pyclass]` now supports fieldless enums. It also provide default implementation for `__repr__`, `__int__`, and `__richcmp__` for enums.
```

Comment:
Might be tempted to skip the detail a bit (interested users can find it in the guide).

```suggestion
- Enable `#[pyclass]` for fieldless (aka C-like) enums. [#2034](https://github.com/PyO3/pyo3/pull/2034)
```